composer.json

```diff
@@ -5,7 +5,8 @@
     "require": {
         "php": ">=7.1",
         "humanmade/hm-gtm": "~2.0.7",
-        "altis/aws-analytics": "~4.5.9"
+        "altis/aws-analytics": "~4.5.9",
+        "altis/altis-analytics-integration-segment": "~1.0.0"
```

Comment:
Will need to add the install overrides, and I'd much prefer if the package name doesn't duplicate "altis", e.g. just `altis/analytics-integration-segment` - will note on the package PR too.